runtime/doc/filetype.txt

```diff
@@ -553,6 +553,23 @@ Options:
 For further discussion of fortran_have_tabs and the method used for the
 detection of source format see |ft-fortran-syntax|.
 
+
+FREEBASIC						*ft-freebasic-plugin*
+
+This plugin aims to treat the four FreeBASIC dialects, "fb", "qb", "fblite"
+and "deprecated", as distinct languages.
+
+The dialect will be set to the first name found in g:freebasic_forcelang, any
+#lang directive or $lang metacommand in the file being edited, or finally
+g:freebasic_lang.  These global variables conceptually map to the fbc options
+-forcelang and -lang.  If no dialect is explicitly specified "fb" will be
+used.
+
+For example, to set the dialect to a default of "fblite" but still allow for
+any #lang directive overrides, use the following command: >
+let g:freebasic_lang = "fblite"
```

Comment:
Needs an indent

Reply:
Also some extra spacing was added in this rt update between the headings

Reply:
Weird. I copied and pasted the whole block...

Reply:
And the extra spacing was deliberate to replicate the spacing in the runtime update?

Reply:
Update also added extra newlines before the GPROF and MAIL headings

Reply:
oooh... I'm blind.

Reply:
How random, though...